test/distributed/test_c10d_nccl.py

```diff
@@ -1031,9 +1031,12 @@ def test_nccl_dist_backend_error(self):
         self._create_process_group_nccl(store, self.opts())
 
         # Both rank 0 and 1 will use the same CUDA device resulting in ncclInvalidUsage
-        with self.assertRaises(dist.DistBackendError):
+        with self.assertRaises(dist.DistBackendError) as cm:
             dist.broadcast(torch.tensor([1, 2, 3]).cuda(), 0)
 
+        self.assertIsInstance(cm.exception, RuntimeError)
+        self.assertIsInstance(cm.exception, Exception)
```

Comment:
```
>>> re = RuntimeError()
>>> isinsstance(re, Exception)
True
```

Is the second `assertIsInstance` redundant?

Reply:
good point